dvc/remote/gdrive.py

```diff
@@ -362,11 +319,7 @@ def _get_remote_drive_id(self, remote_id):
     def _get_cached_remote_ids(self, path):
         if not path:
             return [self._bucket]
-        if self._cache_initialized:
```

Comment:
this logic is important, we don't want to initialize cache if it's not initialized yet

Reply:
cache property init makes a call to self._get_remote_id itself to get root id. I'm not sure who would it behave with these changes